addons/l10n_es_pos/static/src/overrides/models/pos_store.js

```diff
@@ -19,7 +19,7 @@ patch(PosStore.prototype, {
 
         // We need to update the partner list after the pos config is loaded
         if (this.config.is_spanish && !this.config.simplified_partner_id) {
-            await this._loadPartners([this.config.raw.simplified_partner_id]);
+            await this._loadPartners([this.config._raw.simplified_partner_id]);
```

Comment:
I think there is no need for the changes from `.raw` to `._raw` since there is the `raw` getter.

Reply:
Same thing but in a faster way